src/source.rs

```diff
@@ -1,20 +1,21 @@
 #![warn(missing_docs)]
 
+use std::path::PathBuf;
 use std::sync::Arc;
 use std::time::Duration;
 
 use crate::shared::{self, prost_timestamp_from_utc};
-use crate::source::sourcer::source_server::{Source, SourceServer};
+use crate::source::sourcer::source_server::Source;
 use crate::source::sourcer::{
     AckRequest, AckResponse, PendingResponse, ReadRequest, ReadResponse, ReadyResponse,
 };
 use chrono::{DateTime, Utc};
 use tokio::sync::mpsc::{self, Sender};
+use tokio::sync::oneshot;
 use tokio_stream::wrappers::ReceiverStream;
-use tonic::transport::Server;
 use tonic::{async_trait, Request, Response, Status};
 
-use self::sourcer::{partitions_response, PartitionsResponse};
+use self::sourcer::{partitions_response, source_server, PartitionsResponse};
 
 mod sourcer {
```

Comment:
Renamed the module to `proto`. The full path for this would be `crate::source::proto` (though we are not exporting it). Its always namespaced under `source` or `map` or similar. 

Also, for anything from this module, I'm using full name resolution starting from `proto::` everywhere.
Eg:
```rust
#[async_trait]
impl<T> proto::source_server::Source for SourceService<T>
where
    T: Sourcer + Send + Sync + 'static,
{
}
```